test/App.test.jsx

```diff
@@ -0,0 +1,72 @@
+import { render, cleanup } from '@testing-library/react';
+import React from 'react';
+import { expect, it, vi, afterEach, beforeEach, describe } from 'vitest';
+import { ThemeProvider } from '@mui/material/styles';
+import { useSession, SessionProvider } from '@inrupt/solid-ui-react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import flushPromises from './testHelpers';
+
+import theme from '../src/theme';
+
+import App from '../src/App';
+
+const AppWithContexts = () => (
```

Comment:
This seems to be the recommended way to test React Contexts with React Testing Library.
https://testing-library.com/docs/example-react-context/